EIPS/eip-2565.md

```diff
@@ -4,34 +4,48 @@ title: Repricing of the EIP-198 ModExp precompile
 author: Kelly Olson (@ineffectualproperty), Sean Gulley (@sean-sn), Simon Peffers (@simonatsn), Justin Drake (@justindrake), Dankrad Feist (@dankrad)
 discussions-to: https://ethereum-magicians.org/t/big-integer-modular-exponentiation-eip-198-gas-cost/4150
 status: Last Call
-review-period-end: 2020-07-31
+review-period-end: 2020-10-09
 type: Standards Track
 category: Core
 created: 2020-03-20
 requires: 198
 ---
 
 ## Simple Summary
-The [EIP-198](https://eips.ethereum.org/EIPS/eip-198) ‘big integer modular exponentiation’, or `ModExp`, precompile is currently overpriced. Re-pricing this precompile will enable more cost efficient verification of RSA signatures, verifiable delay functions (VDFs), primality checks, and more.
+This EIP reprices the [EIP-198](./eip-198.md) ‘big integer modular exponentiation’, or `ModExp`, precompile.
```

Comment:
```suggestion
Defines the gas cost of the `ModExp` (`0x00..05`) precompile.
```
As mentioned previously, we want this entire EIP to stand almost entirely on its own.  It still needs to have a dependency on 198 for the definition of what the ModExp precompile does, but it shouldn't depend on 198 for any of the gas cost stuff being proposed here.